src/spikeinterface/extractors/neoextractors/neobaseextractor.py

```diff
@@ -260,16 +260,17 @@ def __init__(
         units = signal_channels["units"]
 
         # mark that units are V, mV or uV
+        standard_units_and_additional_gains = {"V": 1e6, "Volt": 1e6, "Volts": 1e6, "mV": 1e3, "uV": 1.0}
         self.has_non_standard_units = False
-        if not np.all(np.isin(units, ["V", "Volt", "mV", "uV"])):
+        if not np.all(np.isin(units, list(standard_units_and_additional_gains.keys()))):
             self.has_non_standard_units = True
+            warnings.warn(
```

Comment:
I don't think this warning is very informative as it is. It would not be clear for the user what standard means and which ones are the non-standard. 

I think the warning (if you want to add it, otherwise I can add it into another PR) should indicate which channels have units that are not recognized by spikeinterface, the units, and the recognized units.

Reply:
Yeah, my idea, is that the users will know which channels might be fishy and which units we automatically assign.